src/main/java/eu/dissco/backend/service/MasJobRecordService.java

```diff
@@ -101,7 +101,7 @@ public Map<String, String> createMasJobRecord(Set<MachineAnnotationServiceRecord
             orcid))
         .toList();
     masJobRecordRepository.createNewMasJobRecord(masJobRecordList);
-    return masJobRecordList.stream().collect(Collectors.toMap(MasJobRecord::masId, MasJobRecord::jobId));
+    return masJobRecordList.stream().collect(Collectors.toMap(MasJobRecord::masId, mjr -> mjr));
```

Comment:
I think `mjr -> mjr` could be `Function.identity()`. Reading this post: https://stackoverflow.com/questions/28032827/java-8-lambdas-function-identity-or-t-t
Could be either way do what you think is best, although I think I mostly used `Function.identity()`

Reply:
oh I like that, thanks!